source/_components/nissan_leaf.markdown

```diff
@@ -48,7 +48,7 @@ nissan_connect:
   description: If your car has the updated head unit (NissanConnect rather than Carwings) then the location can be aquired and exposed via a device tracker. If you have a pre-2014 24 kWh Leaf then you will have Carwings and this should be set to false.
   required: false
   type: boolean
-scan_interval:
+update_interval:
```

Comment:
It would be good to change this in the source to `scan_interval`. We're in the process of moving all other similar terms over to that name.

Reply:
That would require an update to the component. While I do agree with you @MartinHjelmare, this update reflects the code changes in the parent PR. Considering that, I would suggest merging this in for now?

Reply:
Yes @frenck! :+1: My comment was more a for the future thing. Sorry about not being clear.